UnityProject/Assets/Scripts/ScriptableObjects/Research/DefaultTechwebData.cs

```diff
@@ -19,22 +19,26 @@ public void GenerateDefaultData()
 		{
 
 			string jsonData = JsonConvert.SerializeObject(technologies);
-			string path = $"{Application.persistentDataPath}/GameData/Research/";
-			string fileName = "TechwebData.json";
+			string path = Path.Combine(Application.streamingAssetsPath, "TechWeb");
+
+
 			Debug.Log(jsonData);
 
 			if (Directory.Exists($"{path}") == false)
 			{
 				Debug.Log($"{path} not found, making one..");
 				Directory.CreateDirectory(path);
 			}
-			if (File.Exists($"{path}{fileName}"))
+
+			path = Path.Combine(path, "TechwebData.json");
```

Comment:
use constants for these strings

Reply:
cant use constants for these strings as they aren't constant. Paths.Combine will produce a different string depending on the OS